lib/mutant/mutation.rb

```diff
@@ -9,34 +9,37 @@ class Mutation
     CODE_DELIMITER = "\0"
     CODE_RANGE     = (0..4).freeze
 
-    def initialize(subject, node)
-      super(subject, node)
-
-      @source         = Unparser.unparse(node)
-      @code           = sha1[CODE_RANGE]
-      @identification = "#{self.class::SYMBOL}:#{subject.identification}:#{code}"
-      @monkeypatch    = Unparser.unparse(subject.context.root(node))
```

Comment:
This work used to block the main thread. But its specific to an individual mutation. Moving unique to a mutation work into the killforks is key to improve concurrency.